util.py

```diff
@@ -1,7 +1,7 @@
 import re
 from typing import List, Tuple, Union
 
-REGEX_PREFIX = {"RJ": r"[Rr][Jj][0-9]{6}", "RG": r"[Rr][Gg][0-9]{5}"}
+REGEX_PREFIX = {"RJ": r"[Rr][Jj][0-9]{8}", "RG": r"[Rr][Gg][0-9]{5}"}
```

Comment:
Maybe using `r"[Rr][Jj][0-9]{6,8}"` instead to support product page with still 6 digit RJ code
6 digit: https://www.dlsite.com/maniax/work/=/product_id/RJ440148.html
8 digit: https://www.dlsite.com/maniax/work/=/product_id/RJ01004019.html

https://regex101.com/r/XrZTYT/1

Reply:
Either works, but I followed the regular expression pattern. https://github.com/fangchunxi1999/pyDLSite/pull/3/commits/daca666b28da98c383ee6ba93f09e93baba8ef24
thanks.